test/terraform_basic_example_test.go

```diff
@@ -21,6 +21,8 @@ func TestTerraformBasicExample(t *testing.T) {
 		Vars: map[string]interface{}{
 			"example": expectedText,
 		},
+
+		NoColor: true,
```

Comment:
+1 on adding this to something that goes through automated tests. However, instead of adding it here, could you add it to [apply_test.go](https://github.com/gruntwork-io/terratest/blob/master/modules/terraform/apply_test.go) and actually update one of those tests to look at the stdout/stderr returned by the `Apply` method and ensure it doesn't contain any of the terminal color codes?